packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -221,15 +222,16 @@ export const Dialog: ComponentWithAs<'div', DialogProps> &
   });
 
   React.useEffect(() => {
+    const target = contentRef?.current;
     if (open) {
-      lockBodyScroll(context.target);
+      disableBodyScroll(target);
     }
     return () => {
       if (open) {
-        unlockBodyScroll(context.target);
+        clearAllBodyScrollLocks(target);
       }
     };
-  }, [context.target, open]);
+  }, [open]);
```

Comment:
Disclaimer: I'm not 100% sure about this but I think it should be safer to use the ref as a dependency 

```suggestion
  }, [open, contentRef]);
```

Reply:
I don't think so, if we have to it should be `contentRef.current`